plugins/delimiter-extractor/src/parser.ts

```diff
@@ -35,7 +32,10 @@ export async function parseBuffer(
         ],
         dynamicTyping: options?.dynamicTyping || false,
         header: false,
-        skipEmptyLines: options?.skipEmptyLines || 'greedy',
+        skipEmptyLines:
+          options?.skipEmptyLines || options?.headerSelectionEnabled
+            ? false
+            : 'greedy',
```

Comment:
I think this should be: 

```suggestion
        skipEmptyLines: options.headerSelectionEnabled
          ? false
          : options?.skipEmptyLines || 'greedy',
```

Reply:
agreed.